.github/workflows/build.yml

```diff
@@ -0,0 +1,96 @@
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - '*.md'
+      - 'LICENSE'
+    branches:
+      - master
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - 'LICENSE'
+    branches:
+      - master
+  merge_group:
```

Comment:
This doesn't do anything, right?

Reply:
This means that this CI will trigger on any merge_group events.
Ref: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue

We could add strict to only trigger CI merge_group only targeting `master` branch, but since `merge_group` could only be turn on in branch protection rules, so I think we are fine here.

Will add comments to explain this.

Reply:
updated